config/parallel.json

```diff
@@ -0,0 +1,14 @@
+
+{ 
```

Comment:
Should have different config (2 different browsers, one mobile for example) to run the parallel tests on, rather than running all on same config.

Reply:
Fixed https://github.com/browserstack/pytest-browserstack/pull/2/commits/4eb3e554a7d2cdf532ee88f11b0d3c09520b27db